java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -1206,4 +1206,56 @@ public void type(BaseClient client) {
         assertTrue("zset".equalsIgnoreCase(client.type(zsetKey).get()));
         assertTrue("stream".equalsIgnoreCase(client.type(streamKey).get()));
     }
+
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void rpushx(BaseClient client) {
+        String key1 = UUID.randomUUID().toString();
+        String key2 = UUID.randomUUID().toString();
+        String key3 = UUID.randomUUID().toString();
+
+        assertEquals(1, client.rpush(key1, new String[] {"0"}).get());
+        assertEquals(4, client.rpushx(key1, new String[] {"1", "2", "3"}).get());
+        assertArrayEquals(new String[] {"0", "1", "2", "3"}, client.lrange(key1, 0, -1).get());
+
+        assertEquals(0, client.rpushx(key2, new String[] {"1"}).get());
+        assertArrayEquals(new String[0], client.lrange(key2, 0, -1).get());
+
+        // Key exists, but it is not a list
+        assertEquals(OK, client.set(key3, "bar").get());
+        ExecutionException executionException =
+                assertThrows(ExecutionException.class, () -> client.rpushx(key3, new String[] {"_"}).get());
+        assertTrue(executionException.getCause() instanceof RequestException);
+        // empty element list
+        executionException =
```

Comment:
Don't think this is worth testing, you are not sending enough arguments to Glide Core which should be trivial.

Reply:
Error comes from redis.
Will wait for other opinions to remove ot not